__mocks__/mapDefinitions/FullTranscriptMapDefinition.ts

```diff
@@ -45,7 +45,7 @@ ns0:Root:
     Trips:
       $for(/ns0:Root/Looping/VehicleTrips/Trips, $i):
         Trip:
-          VehicleRegistration: /ns0:Root/Looping/VehicleTrips/Vehicle[is-equal(VehicleID, /ns0:Root/Looping/VehicleTrips/Trips[$i]/VehicleId)]/VehicleRegistration
+          VehicleRegistration: /ns0:Root/Looping/VehicleTrips/Vehicle[is-equal(VehicleId, /ns0:Root/Looping/VehicleTrips/Trips[$i]/VehicleId)]/VehicleRegistration
```

Comment:
This is currently testing nothing in it's use case: `is-equal(VehicleId, /ns0:Root/Looping/VehicleTrips/Trips[$i]/VehicleId)`

Both VehicleId are referring to the Trips vehicle id. I wonder if it should be: `is-equal(VehicleId, /ns0:Root/Looping/VehicleTrips/Vehicle[$i]/VehicleID)` instead.

Reply:
Will follow up w/ backend to revisit Transcript map definition for cases such as these